db/migrations/manual/20251231-runout.sql

```diff
@@ -0,0 +1,20 @@
+-- This is a manual DB wrangle to convert Dec CCLF attribution into runouts
+-- We will undo this and convert back to Dec CCLF once the runout for 2024 is received
+
+BEGIN;
+DO $$
+
+DECLARE cclfids integer ARRAY;
+BEGIN
+cclfids := ARRAY(SELECT id FROM cclf_files WHERE timestamp > '2025-11-30' AND type = 0 AND aco_cms_id NOT LIKE ALL(ARRAY['A999%', 'V99%', 'E999%']));
```

Comment:
Are there any other aco_cms_id values that should be added to this list? I assume these are test ACOs, and I think  a recent discussion also mentioned the patterns `%TEST%` and `%DA999%`